### PR TITLE
Add compression settings view

### PR DESCRIPTION
**Pull request by gayyappan**
Add informational view that lists the settings
used while enabling compression on a hypertable.

### CODE REVIEW

File: sql/views.sql [hunk 1]
```diff
@@ -259,5 +259,40 @@ CREATE OR REPLACE VIEW timescaledb_information.data_node AS
...
+SELECT
+   ht.schema_name,
+   ht.table_name,
+   segq.attname,
+   segq.segmentby_column_index,
+   segq.orderby_column_index,
+   segq.orderby_asc,
+   segq.orderby_nullsfirst 
+FROM
+   (
```
Comment (svenklemm) [inline sql/views.sql:274]: should get rid of subselect and also run pg_format on the query

Comment (gayyappan) [reply sql/views.sql:274]: don't care for the sub-select myself. But is there any specific reason why it should not be used (didn't see any plan issues).

File: tsl/test/sql/compression.sql [hunk 2]
```diff
@@ -29,6 +29,7 @@ SET timescaledb.enable_transparent_decompression to OFF;
 select id, schema_name, table_name, compressed, compressed_hypertable_id from
 _timescaledb_catalog.hypertable order by id;
 select * from _timescaledb_catalog.hypertable_compression order by hypertable_id, attname;
+select * from timescaledb_information.compression_settings ;
```
Comment (svenklemm) [inline tsl/test/sql/compression.sql:32]: needs ORDER BY clause

Comment (gayyappan) [reply tsl/test/sql/compression.sql:32]: The view already has ORDER BY.

**Comment by gayyappan** [comment]
> Looks good once you add the ORDER BY requested by Sven.

The view definition already has the ORDER BY.

**Pull request merged**